docs/getting-started-interfaces.md

```diff
@@ -0,0 +1,59 @@
+# How do I get started, what interfaces can I use?
+
+To interact with a large language model (LLM) it is recommend that you utilize one of the pre-existing interfaces that have been built by the open-source community.
+
+There are a few interface options based on your particular needs:
+
+##### text-generation-webui (aka Oobabooga)
+
+![UI Image](https://raw.githubusercontent.com/oobabooga/screenshots/main/galactica.png)
```

Comment:
Page content looks good, The images need to be set to the same size or aspect ratio though. They show up pretty large on the page. I'm not sure if we have a smaller icon and you can click on it to see the full image